ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/mpls.yml

```diff
@@ -0,0 +1,73 @@
+---
+mpls:
+  ip: true
+  ldp:
+    interface_disabled_default: true
+    router_id: 192.168.1.1
+    shutdown: false
+    transport_address_interface: Loopback0
+  icmp:
+    fragmentation_needed_tunneling: true
+  rsvp:
+    refresh:
+      interval: 3
+    authentication:
+      password_indexes:
+        - index: 766
+          password_type: 7
+          password: 13361E3F1B20011A2A173B
+        - index: 55
+          password_type: '7'
+          password: 14241B062114062E142900260B1D23133B
+        - index: 999
+          password: simplePassword
+    fast_reroute:
+      reversion: local
+      bypass_tunnel_optimization_interval: 65535
+    srlg:
+      enabled: true
+      strict: true
+    graceful_restart:
+      role_speaker:
```

Comment:
if possible to improve the coverage for role_speaker and role_helper within these three host then please improve it. I can see some partial tests are missing for the same.

Reply:
Done!